### PR TITLE
Default values for arguments

### PR DESCRIPTION
**Pull request by marius-baseten**
## :microscope: Testing

Run integration test locally using branch `git+https://github.com/basetenlabs/truss.git@marius/default-values` as requirement for test chain.

### CODE REVIEW

File: truss-chains/tests/chains_test.py [hunk 1]
```diff
@@ -25,12 +25,45 @@ def test_chain():
 
         url = service.run_url.replace("host.docker.internal", "localhost")
 
+        # Call without providing values for default arguments.
+        response = requests.post(url, json={"length": 30, "num_partitions": 3})
+        print(response.content)
```
Comment (bolasim) [inline truss-chains/tests/chains_test.py:30]: is this needed?

File: truss-chains/tests/chains_test.py [hunk 2]
```diff
@@ -25,12 +25,45 @@ def test_chain():
...
-            url, json={"length": 30, "num_partitions": 3}, stream=True
+            url,
+            json={
+                "length": 30,
+                "num_partitions": 3,
+                "pydantic_default_arg": {
+                    "parts": ["marius"],
+                    "part_lens": [3],
+                },
+                "simple_default_arg": ["bola"],
```
Comment (bolasim) [inline truss-chains/tests/chains_test.py:52]: LOL

File: truss-chains/tests/itest_chain/itest_chain.py [hunk 3]
```diff
@@ -81,7 +87,8 @@ async def run_remote(
             ),
             extra_arg=123,
         )
+        print(pydantic_default_arg, simple_default_arg)
```
Comment (bolasim) [inline truss-chains/tests/itest_chain/itest_chain.py:90]: nit: print

**Review by bolasim — APPROVED** [review]

File: truss-chains/tests/chains_test.py (hunk 1 above)
Comment (marius-baseten) [reply truss-chains/tests/chains_test.py:30]: Not needed, but this is a test, so I feel it's fine to leave print statements like this (they will be filtered anyway **unless** the test fails).

File: truss-chains/tests/chains_test.py (hunk 2 above)
Comment (marius-baseten) [reply truss-chains/tests/chains_test.py:52]: Somebody put this already here: https://docs.baseten.co/chains/guide#test-a-chain-locally (I don't know who, but it wasn't me...)

**Pull request merged**